routing_model/index.html

```diff
@@ -746,7 +746,7 @@ <h2>Process Adult/Elder promotion and demotion needed from last check</h2>
   </div>
   <div class="mermaid">
       graph TB
-      ProcessElderChange["ProcessElderChange<br/>(Take elder changes)"]
+      ProcessElderChange["ProcessElderChange<br/>(Take elder changes)<br/>(shared state)"]
```

Comment:
I'm not too convinced about these "shared state" annotations... What do they mean exactly? I would prefer to explicitely mention which state is shared (or leave it to the more detailed explanations)

Reply:
This refers to the chain.rs (i.e all information that we computed from consensused information).
We cannot update this if we are not triggered by a consensus event.
We probably want an explanation somewhere.

Reply:
Ah! I see! OK, we can make it clear by documenting it. Maybe even come up with a better name for it such as: `in order` or so (we can do this in a different PR, though)